README.md

```diff
@@ -67,10 +67,10 @@ response in a text view.
 You can build the application by running:
 
 ```
-$ bazel build //ios-app:GetUrl
+$ bazel build ios-app --ios_sdk_version=8.4
```

Comment:
Why this change? Do users need to change this to the version they want to build with?